app/scripts/index.js

```diff
@@ -147,7 +148,7 @@ app.run(function ($rootScope, $location, $anchorScroll, $window) {
       // check if session has expired
       let currentTime = new Date()
       currentTime = currentTime.toISOString()
-      if (currentTime >= consoleSession.expires) {
+      if (currentTime >= consoleSession.expires || !$cookies.get("koa.sess")) {
```

Comment:
```suggestion
      if (!$cookies.get("koa.sess")) {
```
Let's remove the consoleSession ;)

Reply:
+ cookie attribute "httpOnly" should be set to true, so that the frontend JS cannot access it. This will disallow 3rd party JS from stealing the session cookie.

To know if the user is initially logged in or not we could use the loggedIn localstorage which should be populated on initial load.

Reply:
done :)